guides/user_roles.md

```diff
@@ -59,7 +63,7 @@ defmodule MyAppWeb.EnsureRolePlug do
   def init(config), do: config
 
   @doc false
-  @spec call(Conn.t(), atom() | list(atom() | String.t())) :: Conn.t()
+  @spec call(Conn.t(), atom() | binary() | [atom()] | [binary()]) :: Conn.t()
```

Comment:
I think we could even do 
` @spec call(Conn.t(), atom() | binary() | [atom() | binary()]) :: Conn.t()`

Reply:
I was thinking about that, but I think it might be better to infer that it should be a list of either binaries or atoms and not mixed even if it's not how the code works. Not sure.